Algorithms.Tests/Strings/GeneralStringAlgorithmsTests.cs

```diff
@@ -20,5 +20,14 @@ public static void MaxCountCharIsObtained(string text, char expectedSymbol, int
             Assert.AreEqual(expectedSymbol, symbol);
             Assert.AreEqual(expectedCount, count);
         }
+
+        [Test]
+        [TestCase("bluh", 4, new string[]{"b","l","u","h"})]
```

Comment:
Codacy found an issue: [Remove the array type; it is redundant.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6295708)